multi-node/aws/pkg/config/templates/cloud-config-controller

```diff
@@ -56,6 +69,31 @@ coreos:
         [Install]
         WantedBy=multi-user.target
 
+{{ if eq .ContainerRuntime "rkt" }}
+    - name: rkt-api.service
+      command: start
+      enable: true
+      content: |
+        [Service]
+        ExecStart=/usr/bin/rkt api-service
+        Restart=always
+        RestartSec=10
+
+    - name: load-rkt-stage1.service
+      command: start
+      content: |
+        [Unit]
+        Description=Load rkt stage1 images
+        Documentation=http://github.com/coreos/rkt
+        Requires=network-online.target
+        After=network-online.target
+        RequiredBy=kubelet.service rkt-api.service
+        Before=kubelet.service rkt-api.service
```

Comment:
For the `RequiredBy/Before` lines, just list rkt-api.service here, and move the `kubelet.service` dependency injection up to `rkt-api.service`.